examples/data-sources/hpegl_vmaas_passive_monitor/data-source.tf

```diff
@@ -0,0 +1,6 @@
+ (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_passive_monitor" "tf_lb_passive" {
```

Comment:
u have one resource for monitor. why multiple data sources for monitor ?

Reply:
removed